### DISCUSSION TITLE
Cumulative check function for deployed environments

### DISCUSSION THREAD

**Discussion by adoc-github** [QualiTorque/qualitorque.github.io#188] [Ideas]
I would like you to be able to check the cumulative total of deployed environments.

Each license plan limits the number of environments a user can deploy.
Users would then want to know how many environments they have deployed.
Therefore, I think it would be good to be able to check the cumulative total of deployed environments on the Torque screen.
do you like this idea? Please consider implementing it.

**Comment by TomerAdmon** [comment]
Hi @adoc-github 
Licensing usage over time can be found under the Torque management area, in the "Account & Billing" tab. Here is an example:

Is that what you are looking for? Feel free to suggest other graphs and we will be happy to consider adding them to our roadmap.

![image](https://user-images.githubusercontent.com/12639815/209686217-70d906fe-d31a-4bf2-9765-304ea1d9b5cd.png)

**Reply by adoc-github** [reply]
Hi @TomerAdmon 

That information is not what I want to know.
I understand that the display is the number of concurrently deployed environments by day.
The information I want to know is the total number of environments deployed each month, or how many environments can be deployed.
For example, in the case of the "PROFESSIONAL" plan, the maximum number of environments that can be deployed is 500 per month.
Therefore, I think it would be nice if the user could confirm information such as "300 environments have been deployed out of the maximum of 500 environments per month".
I don't think that such confirmation is possible with Torque today.
Therefore, I would like you to implement a UI that allows the user to confirm them.

**Reply by TomerAdmon** [reply]
Thanks @adoc-github.
This is already in our product plan, we will keep you updated with the release plan.

**Reply by adoc-github** [reply]
Thanks @TomerAdmon 
I'll be waiting for updates on the release plan.
Also, I would like to know when we can get an update on the release plan.
Is that possible?

**Reply by TomerAdmon** [reply]
Sure @adoc-github, 
Once we have an update we will share It with you directly with the help of @LeeorV

I will go ahead and close this discussion for now.
